packages/build-utils/src/fs/read-config-file.ts

```diff
@@ -25,12 +25,16 @@ export async function readConfigFile<T>(
 
     if (data) {
       const str = data.toString('utf8');
-      if (name.endsWith('.json')) {
-        return JSON.parse(str) as T;
-      } else if (name.endsWith('.toml')) {
-        return (toml.parse(str) as unknown) as T;
-      } else if (name.endsWith('.yaml') || name.endsWith('.yml')) {
-        return yaml.safeLoad(str, { filename: name }) as T;
+      try {
+        if (name.endsWith('.json')) {
+          return JSON.parse(str) as T;
+        } else if (name.endsWith('.toml')) {
+          return toml.parse(str) as unknown as T;
+        } else if (name.endsWith('.yaml') || name.endsWith('.yml')) {
+          return yaml.safeLoad(str, { filename: name }) as T;
+        }
+      } catch (error: unknown) {
+        console.log(`Error while parsing config file: "${name}"`);
```

Comment:
Should it not tell the user why it was trying to do this, and/or why it can't continue the build now?
Do we not want to print the actual error that was thrown as a way for the user to debug/see what's wrong?

Reply:
It will continue the build. The idea is that whatever tool uses the config file later will have a hard error such as `pnpm install`